app/conf/consumers.json

```diff
@@ -0,0 +1,44 @@
+[
+  {
+    "name": "es",
+    "url": "http://elasticsearch-consumer:9099/",
+    "resources": {
+      "elasticsearch": {
+        "id": "elasticsearch-id-1",
+        "name": "ES Instance",
+        "url": "http://elasticsearch:9200"
+      },
+      "kibana": {
+        "id": "kibana-id-1",
+        "name": "Kibana Instance",
+        "url": "http://kibana:5601/kibana-app"
+      }
+    },
+    "subscription": {
+      "id": "subscription-id-1",
+      "name": "Subscription",
+      "topic_pattern": "*",
+      "topic_options": {
+          "masking_annotation": "@aether_masking",
+          "masking_levels": ["public", "private"],
+          "masking_emit_level": "public",
+          "filter_required": false
+      },
+      "es_options": {
```

Comment:
if the resource is `elasticsearch`, why are the options `es_options`?

Reply:
`es_options` is on the subscription object, and unique to the es consumer job function